network-static-2-httpks.ks.in

```diff
@@ -0,0 +1,68 @@
+#test name: network-static-2
+#various combinations of activation in initramfs and stage 2
+url @KSTEST_URL@
+install
+# activated in initramfs with dhcp, reactivated in stage 2
+network --device=ens4 --bootproto static --ip @KSTEST_STATIC_IP1@ --netmask @KSTEST_STATIC_NETMASK@ --gateway @KSTEST_STATIC_GATEWAY@
+# activated in initramfs with dhcp, not reactivated in stage 2
+network --device=ens5 --bootproto static --ip @KSTEST_STATIC_IP2@ --netmask @KSTEST_STATIC_NETMASK@ --gateway @KSTEST_STATIC_GATEWAY@
+# not activated in initramfs with dhcp, reactivated in stage 2
+network --device=ens6 --bootproto static --ip @KSTEST_STATIC_IP3@ --netmask @KSTEST_STATIC_NETMASK@ --gateway @KSTEST_STATIC_GATEWAY@ --activate
+# not activated in initramfs with dhcp, not reactivated in stage 2
+network --device=ens7 --bootproto static --ip @KSTEST_STATIC_IP4@ --netmask @KSTEST_STATIC_NETMASK@ --gateway @KSTEST_STATIC_GATEWAY@
+
+bootloader --timeout=1
+zerombr
+clearpart --all
+autopart
+
+keyboard us
+lang en
+timezone America/New_York
+rootpw qweqwe
+shutdown
+
+%packages
+%end
+
+%post --nochroot
+
+@KSINCLUDE@ post-nochroot-lib-network.sh
+
+check_gui_configurations ens3 ens4 ens5 ens6 ens7
+
+%end
+
+%post
+
+@KSINCLUDE@ post-lib-network.sh
+
+check_device_connected ens3 yes
+
+check_device_ifcfg_value ens4 IPADDR @KSTEST_STATIC_IP1@
+check_device_connected ens4 yes
+check_device_ipv4_address ens4 @KSTEST_STATIC_IP1@
+
+check_device_ifcfg_value ens5 IPADDR @KSTEST_STATIC_IP2@
+check_device_connected ens5 yes
+
+check_device_ifcfg_value ens6 IPADDR @KSTEST_STATIC_IP3@
+check_device_connected ens6 yes
+check_device_ipv4_address ens6 @KSTEST_STATIC_IP3@
+
+check_device_ifcfg_value ens7 IPADDR @KSTEST_STATIC_IP4@
+if [[ "@KSTEST_OS_NAME@" == "rhel" ]]; then
+    check_device_connected ens7 no
+else
+    # Fedora autoactivates default connections (Wired Connection 2)
+    # if it doesn't find ifcfg for the device, which is the case for
+    # kickstart in %pre (unlike normal kickstart where the ifcfg file
+    # is created in initramfs.
```

Comment:
Will add, this (with the related code) I will move to network-static-2-pre.ks.in where it really belongs.